process-phoenix/src/main/java/com/jakewharton/processphoenix/ProcessPhoenix.java

```diff
@@ -54,14 +56,14 @@ public static void triggerRebirth(Context context) {
   }
 
   /**
-   * Call to restart the application process using the specified intent.
+   * Call to restart the application process using the specified intents.
    * <p>
    * Behavior of the current process after invoking this method is undefined.
    */
-  public static void triggerRebirth(Context context, Intent nextIntent) {
+  public static void triggerRebirth(Context context, Intent... nextIntents) {
```

Comment:
This is a binary incompatible change. Please retain a single argument version and delegate to this method with `new Intent[] { nextIntent }`.